### PR TITLE
Song library styling

### PR DESCRIPTION
**Pull request by monshan**
## What does this PR do (summary of changes)?

- Applies CSS grid and basic alignment styling to the SongLibrary and SongCard components
- Adds one breakpoint at 1025px

## How should it be tested?

 - Can observe stylistic changes on the DOM

## Future Iterations:

 1. The cards may need to be smaller (both text and spread) to better accomodate smaller screens
 2. Should genres be clickable or interactive?

### CODE REVIEW

**Comment by m1073496** [comment]
[Style the songCard component and Library container component](https://trello.com/c/J061rGoW/59-style-the-songcard-component-and-library-container-component)

**Comment by elizhahn** [comment]
just wondering if you have a card border in there now just for setting margins and alignment, If I remember we didn't want to have borders separating the list of songs?

File: src/App.js [hunk 1]
```diff
@@ -33,21 +34,23 @@ class App extends Component {
...
-      <div className="app">
+      <div className="App">
         <Route exact path="/">
           <h1>CarryOkay</h1>
           <p>Hello friend 😬</p>
           <Navigation class="home-nav" />
         </Route>
         <Route path="/mysongs">
-          <p>Made it to mysongs</p>
+          <h1>My Songs</h1>
```
Comment (m1073496) [inline src/App.js:44]: Thanks for this change, I think this was failing in the tests bc the title was wrong

File: src/components/SongLibrary/SongLibrary.css [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+
+.library > * {
+  grid-column: span 1;
+}
+
+@media (max-width: 1025px) {
+  .library {
+    grid-template-columns: minmax(260px, 1fr);
+  }
+}
```
Comment (m1073496) [inline src/components/SongLibrary/SongLibrary.css:19]: Nice! Yeah I was thinking earlier that we've been designing for the browser so far, even though we initially had a mobile app primarily in mind. Thanks for this!

**Review by m1073496 — APPROVED** [review]
Looks good! Thanks for your work!!

**Pull request merged**